src/install.js

```diff
@@ -99,14 +159,17 @@ async function installSAPUI5() {
   try {
     await prepareFileSystem(libDir, downloadDir)
 
-    const sapui5Archive = await downloadSAPUI5(latestVersionURL, downloadDir)
-    await extractArchive(sapui5Archive, libDir)
+    await downloadSAPUI5(latestVersionURL, downloadDir).then((sapui5Archive) => {
+      extractArchive(sapui5Archive, libDir)
+    }).catch((error) => {
+      console.log(error.message)
+    })
```

Comment:
Why did you introduce a separate `then` and `catch` here? If `downloadSAPUI5` throws an error or rejects the promise, the catch block of the `try, catch` will be executed and print the error.
What is the reason for this change? I might be wrong here as I didn't test it locally. Maybe the catch is required.

Reply:
It was due to my testing of the unexpected archive. It's reverted back